src/main/kotlin/no/nav/familie/ba/sak/dokument/MalerService.kt

```diff
@@ -106,7 +108,7 @@ class MalerService(
                                         enhet: String,
                                         målform: Målform): String {
         val totrinnskontroll = totrinnskontrollService.opprettEllerHentTotrinnskontroll(vedtak.behandling)
-        val etterbetalingsbeløp = etterbetalingsbeløpFraSimulering().takeIf { it > 0 }
+        val etterbetalingsbeløp = etterbetalingsbeløpFraSimulering(vedtak).takeIf { it > 0 }
```

Comment:
```suggestion
        val etterbetalingsbeløp = økonomiService.hentEtterbetalingsbeløp(vedtak).etterbetaling.takeIf { it > 0 }
```

Når det ikke var mer mikkmakk i funksjonen kan vi kanskje kalle de direkte? :) funksjonen var litt en placeholder